res/TensorFlowLiteRecipes/Net_InstanceNorm_006/test.recipe

```diff
@@ -0,0 +1,283 @@
+# same as Net_InstanceNorm_004 but having one element for gamma, beta
```

Comment:
(optinal) I think this file better be described without any other files' explanation becoz it would become its dependency. If 004 is changed, this file should be changed either. But seems rarely happen:)